_posts/release/2025-05-01-accumulo-2.1.4.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Apache Accumulo 2.1.4
+sortableversion: '02.01.04'
+draft: true
+LTM: true
+---
+## About
+
+Apache Accumulo 2.1.4 is a patch release of the 2.1 LTM line. It contains bug
+fixes and minor enhancements. This version supersedes 2.1.3. Users upgrading to
+2.1 should upgrade directly to this version instead of 2.1.3.
+
+## Notable Changes
+
+* {% ghi 5446 %} Many dependencies were updated, to include protobuf-java which
+  was updated from 3.22.0 to 3.25.6 due to a vulnerability. Users may run into
+  a runtime exception added in https://github.com/protocolbuffers/protobuf/pull/20084
+  when using old generated Protobuf files. Users are advised to regenerate their
+  protobuf files using protobuf 25.6 or later. Alternatively, users can set a
+  system property (see #20084 for details) to suppress the exception.
+
+* {% ghi 5073 %} {% ghi 5034 %} Added `accumulo check-accumulo-properties` command that can be
+  run before an instance is initialized to check the properties file.
+
+* {% ghi 5193 %} Added `accumulo admin signalShutdown` command to signal the server
+  process to initiate a graceful shutdown. Compactors will finish the major compaction
+  that they are currently working on, then will exit. Scan Servers will return a busy
+  signal to all clients for new scan requests, and will shutdown when all in-progress
+  scans have closed. Tablet Servers will signal the Manager that they are shutting down,
+  which should prevent assignment of tablets to that server, then they will unload all
+  tablets and shut down. The Monitor, Manager, GarbageCollector, and CompactionCoordinator
+  will shut down also. The last step in the shutdown process for all servers is to remove
+  their lock in ZooKeeper.
+
+* {% ghi 5438 %} Added `accumulo upgrade --prepare` command which should be used after
+  shutting down an instance in preparation for an upgrade. This will check that no Fate
+  transactions exist, delete any ZooKeeper locks for server processes, and prohibit any
+  server processes from being started.
+
+* {% ghi 4898 %} New way to compute bulk load plans (TODO: Needs text)
```

Comment:
TODO

Reply:
This will probably be filled in by Keith in a subsequent PR